tests/test_age_pyramid.py

```diff
@@ -8,31 +8,36 @@
 
 from eds_scikit.datasets.synthetic.person import load_person
 from eds_scikit.plot.data_quality import plot_age_pyramid
+import pandas as pd 
+import numpy as np
 
 data = load_person()
 
+person_with_inclusion_date = data.person.copy()
+person_with_inclusion_date["inclusion_datetime"] = person_with_inclusion_date["birth_datetime"] + pd.to_timedelta(np.random.randint(0, 1000, len(person_with_inclusion_date)), unit='d')
```

Comment:
Nice idea. There's a lot going in this line, could you add some extra steps for readability?
Like so:
```python
N = len(person_with_inclusion_date)
delta_days = pd.to_timedelta(np.random.randint(0, 1000, N), unit='d')
person_with_inclusion_date["inclusion_datetime"] = (
    person_with_inclusion_date["birth_datetime"]
    + delta_days
)
```
Also, let's apply Black formater.